### PR TITLE
Cleaned unused stuff + proper names for turbofire buttons + minor alterations to sublabels

### PR DESCRIPTION
**Pull request by sonninnos**
Also confined mouse cursor to the key area on mouse move.

### CODE REVIEW

File: libretro/nukleargui/app.c [hunk 1]
```diff
@@ -123,24 +116,12 @@ int app_event(int poll)
 
 int app_render(int poll)
 {
-    static int prevpoll=0,prevstate=0,reset_state=0;
-    if(prevpoll!=poll || reset_state){nk_clear(ctx);reset_state=0;}
```
Comment (rsn8887) [inline libretro/nukleargui/app.c]: Do we really want to remove the nk_clear() call that happens after each opening/closing of the keyboard? The nuklear docs say nk_clear should be called. https://rawgit.com/vurtun/nuklear/master/doc/nuklear.html#nuklear/api

File: libretro/nukleargui/app.c [hunk 2]
```diff
@@ -123,24 +116,12 @@ int app_event(int poll)
...
-
-    if(poll==0)
-	app_vkb_handle();
-    else 
-	restore_bgk();
-
+    app_vkb_handle();
     app_event(poll);
```
Comment (rsn8887) [inline libretro/nukleargui/app.c]: change poll to zero

File: libretro/nukleargui/app.c [hunk 3]
```diff
@@ -123,24 +116,12 @@ int app_event(int poll)
...
 
     int state=gui(ctx);
-    if(state==1 && prevstate!=1)reset_state=1;
```
Comment (rsn8887) [inline libretro/nukleargui/app.c]: Don't we want nk_clear calls?

File: libretro/nukleargui/app.c [hunk 4]
```diff
@@ -123,24 +116,12 @@ int app_event(int poll)
...
-
     nk_retro_render(nk_rgba(0,0,0,0));
 
-    prevpoll=poll;
-    prevstate=state;
```
Comment (rsn8887) [inline libretro/nukleargui/app.c]: Maybe leave this in to get nk_clear calls?

File: libretro/nukleargui/app.c (hunk 1 above)
Comment (sonninnos) [reply libretro/nukleargui/app.c]: At the moment it never happens, because app_render is only called with poll 0.

Comment (sonninnos) [reply libretro/nukleargui/app.c]: Oh right, the reset_state is changed on opening/closing the vkbd. I'll test some more.

**Comment by sonninnos** [comment]
OK, I left out the poll but kept the reset, even though it seemed to make no difference.

**Comment by rsn8887** [comment]
Awesome, thanks!

**Pull request merged**